cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/VendorReportApi.java

```diff
@@ -12,6 +12,10 @@
 import java.util.List;
 import java.util.Map;
 
+/**
+ * @deprecated by {@link ResellingReportApi}
+ */
+@Deprecated
 public interface VendorReportApi
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)